nullaway/src/main/java/com/uber/nullaway/AbstractConfig.java

```diff
@@ -349,4 +351,9 @@ public String getErrorURL() {
   public boolean acknowledgeAndroidRecent() {
     return acknowledgeAndroidRecent;
   }
+
+  @Override
+  public boolean allowLibraryModelsOverrideAnnotations() {
```

Comment:
Ok, naming here could be improved. We already allow library models to override annotations (namely, they override restrictive annotations in "unannotated" code - ok, fine, that one was bad naming on our part! 😉 But the point remains...)

I would call this flag:

- `-XepOpt:NullAway:AcknowledgeLibraryModelsOfAnnotatedCode` for the EP flag(I am also fine with "allow" vs "acknowledge", but I am thinking of consistency with some existing flags)
- `acknowledgeLibraryModelsOfAnnotatedCode()` for the method and `acknowledgeLibraryModelsOfAnnotatedCode` for the field (I don't see what "activation" means or adds and it isn't necessary, see `acknowledgeAndroidRecent` for an example of how this looks like)
- "Allow Library Models of annotated code" as the PR's title (side note, generally, imperative mood is recommended in commit subjects - this is part of Uber's Git commit message best practices, but I've also seen it as standard practice elsewhere in most major OSS projects. So, "Allow" instead of "Allowing" or "Allows").

Reply:
Did all the renamings 13eb085ff0ee3b494f8516f143d051a675849b4d
Thank you very much for the note, I will definitely follow this practice in future PRs and commits :)